src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/ConsentShutter.cshtml

```diff
@@ -0,0 +1,28 @@
+﻿@page
+@model FamilyHubs.Referral.Web.Pages.ProfessionalReferral.ConsentShutterModel
+@{
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <form method="post">
+            <fieldset class="govuk-fieldset">
+                <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+                    <h1 class="govuk-fieldset__heading">
+                        Cannot connect family to service
+                    </h1>
+                </legend>
+                <p>You must have permission from the family to give their details to the service.</p>
+                <p class="govuk-body">Those details include:</p>
+                <ul class="govuk-list govuk-list--bullet">
+                    <li>who to contact in the family</li>
+                    <li>ways to contact the family</li>
+                    <li>why the family needs help </li>
+                </ul>
+                <p>You can come back and connect the family to the service when you have permission.</p>
+                <div class="govuk-form-group">
```

Comment:
Do we need this?  (It doesn't have a closing tag.)

Reply:
Removed